### PR TITLE
[Consensus] fix startup time

### PR DESCRIPTION
**Pull request by smnzhu**
`ValidateFlags` needs to be called before `Initialize()`.

### CODE REVIEW

File: cmd/collection/main.go [hunk 1]
```diff
@@ -143,29 +143,28 @@ func main() {
...
-		flags.StringVar(&startupTimeString, "hotstuff-startup-time", cmd.NotSet, "specifies date and time (in ISO 8601 format) after which the consensus participant may enter the first view (e.g 2006-01-02T15:04:05Z07:00)")
+		flags.StringVar(&startupTimeString, "hotstuff-startup-time", cmd.NotSet, "specifies date and time (in ISO 8601 format) after which the consensus participant may enter the first view (e.g (e.g 1996-04-24T15:04:05-07:00))")
 
 		// epoch qc contract flags
 		flags.StringVar(&accessAddress, "access-address", "", "the address of an access node")
 		flags.StringVar(&secureAccessNodeID, "secure-access-node-id", "", "the node ID of the secure access GRPC server")
 		flags.BoolVar(&insecureAccessAPI, "insecure-access-api", true, "required if insecure GRPC connection should be used")
+	}).ValidateFlags(func() error {
+		if startupTimeString != cmd.NotSet {
+			t, err := time.Parse(time.RFC3339, startupTimeString)
```
Comment (zhangchiqing) [inline cmd/collection/main.go:154]: log the startup time here as well?

File: cmd/collection/main.go [hunk 2]
```diff
@@ -143,29 +143,28 @@ func main() {
 			"additional fraction of replica timeout that the primary will wait for votes")
 		flags.DurationVar(&blockRateDelay, "block-rate-delay", 250*time.Millisecond,
 			"the delay to broadcast block proposal in order to control block production rate")
-		flags.StringVar(&startupTimeString, "hotstuff-startup-time", cmd.NotSet, "specifies date and time (in ISO 8601 format) after which the consensus participant may enter the first view (e.g 2006-01-02T15:04:05Z07:00)")
+		flags.StringVar(&startupTimeString, "hotstuff-startup-time", cmd.NotSet, "specifies date and time (in ISO 8601 format) after which the consensus participant may enter the first view (e.g (e.g 1996-04-24T15:04:05-07:00))")
```
Comment (zhangchiqing) [inline cmd/collection/main.go:146]:
```suggestion
		flags.StringVar(&startupTimeString, "hotstuff-startup-time", cmd.NotSet, "specifies date and time (in ISO 8601 format) after which the consensus participant may enter the first view (e.g 2006-04-24T15:04:05-07:00)")
```

**Review by zhangchiqing — APPROVED** [review]

**Review by peterargue — APPROVED** [review]

File: integration/localnet/Makefile [hunk 3]
```diff
@@ -47,8 +47,8 @@ init-short-epochs:
 
 .PHONY: start
 start:
-	docker-compose -f docker-compose.metrics.yml up -d
-	DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker-compose -f docker-compose.nodes.yml up --build -d
+	docker-compose -f docker-compose.metrics.yml up -d --remove-orphans
```
Comment (huitseeker) [inline integration/localnet/Makefile:50]: 🤩

**Review by huitseeker — APPROVED** [review]
LGTM!

**Pull request merged**